README.md

```diff
@@ -2,55 +2,31 @@
 
 # Project Name
 
-> Description the project.
+> Set up a "Hello Microverse" project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS
+- VSC, github
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://www.youtube.com/watch?v=UTga0ARWu2g&t=252s)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+No  setup required
```

Comment:
- [ ] Kindly fill out this section. It's supposed to describe how to setup the project locally so you can write instructions on how to clone the project and run it in the browser etc...